lib/WUI/netdev.c

```diff
@@ -105,110 +110,14 @@ void netdev_get_MAC_address(uint32_t netdev_id, uint8_t mac[OTP_MAC_ADDRESS_SIZE
- * \param[in]     res: Baudrate change result
- * \param[in]     arg: event data, new baudarate
- */
-static void esp_baudrate_changed(espr_t res, void *arg) {
-    if (res != espOK) {
-        _dbg("ESP baudrate change failed !!!");
-        return;
-    }
-
-    _dbg("ESP baudrate change success, reconfiguring UART for %" PRIu32, esp_target_baudrate);
-    esp_reconfigure_uart(esp_target_baudrate);
-}
-
-/**
- * \brief           Event callback function for ESP stack
- * \param[in]       evt: Event information with data
- * \return          \ref espOK on success, member of \ref espr_t otherwise
- */
-static espr_t
-esp_callback_func(esp_evt_t *evt) {
-    switch (esp_evt_get_type(evt)) {
-    case ESP_EVT_AT_VERSION_NOT_SUPPORTED: {
-        esp_sw_version_t v_min, v_curr;
-
-        esp_get_min_at_fw_version(&v_min);
-        esp_get_current_at_fw_version(&v_curr);
-
-        _dbg("Current ESP8266 AT version is not supported by library!");
-        _dbg("Minimum required AT version is: %d.%d.%d", (int)v_min.major, (int)v_min.minor, (int)v_min.patch);
-        _dbg("Current AT version is: %d.%d.%d", (int)v_curr.major, (int)v_curr.minor, (int)v_curr.patch);
-        break;
-    }
-    case ESP_EVT_INIT_FINISH: {
-        esp_set_wifi_mode(ESP_MODE_STA, NULL, NULL, 1);
-        esp_state = NETDEV_UNLINKED;
-        osMessagePut(networkMbox_id, EVT_LWESP_INIT_FINISHED, 0);
-        _dbg("ESP_EVT_INIT_FINISH");
-        break;
-    }
-    case ESP_EVT_RESET: {
-        esp_set_at_baudrate(esp_target_baudrate, esp_baudrate_changed, NULL, 0);
-        _dbg("ESP_EVT_RESET");
-        break;
-    }
-    case ESP_EVT_RESTORE: {
-        _dbg("ESP_EVT_RESTORE");
-        break;
-    }
-    case ESP_EVT_RESET_DETECTED: {
-        esp_reconfigure_uart(ESP_CFG_AT_PORT_BAUDRATE);
-        _dbg("ESP_EVT_RESET_DETECTED");
-        break;
-    }
-    case ESP_EVT_WIFI_GOT_IP: {
-        _dbg("ESP_EVT_WIFI_GOT_IP");
-        esp_state = NETDEV_NETIF_UP;
-        break;
-    }
-    case ESP_EVT_WIFI_CONNECTED: {
-        uint32_t message = EVT_NETDEV_INIT_FINISHED(NETDEV_ESP_ID, 0);
-        _dbg("ESP_EVT_WIFI_CONNECTED");
-        osMessagePut(networkMbox_id, message, 0);
-        break;
-    }
-    case ESP_EVT_WIFI_DISCONNECTED: {
-        _dbg("ESP_EVT_WIFI_DISCONNECTED");
-        esp_state = NETDEV_UNLINKED;
-        break;
-    }
-    case ESP_EVT_SERVER: {
-        _dbg("ESP_EVT_SERVER");
-        break;
-    }
-    case ESP_EVT_WIFI_IP_ACQUIRED: {
-        _dbg("ESP_EVT_WIFI_IP_ACQUIRED");
-        break;
-    }
-    case ESP_EVT_STA_JOIN_AP: {
-        _dbg("ESP_EVT_STA_JOIN_AP");
-        break;
-    }
-    case ESP_EVT_CMD_TIMEOUT: {
-        _dbg("ESP_EVT_CMD_TIMEOUT");
-        break;
-    }
-    default: {
-        _dbg("Unknown ESP message: %d", (int)esp_evt_get_type(evt));
-        break;
-    }
-    }
-    return espOK;
-}
-
 static void alsockets_adjust() {
+    // TODO: Drop alternative sockets
```

Comment:
Yes. Another future PR. And also turn off altcp in the LwIP configuration (probably also in another PR).